src/misc/lv_fs.c

```diff
@@ -493,15 +499,25 @@ const char * lv_fs_get_last(const char * path)
  *   STATIC FUNCTIONS
  **********************/
 
+/**
+ * Checks if the path starts with a drive prefix (i.e. "X:...")
+ * @param path path string (E.g. S:/folder/file.txt)
+ * @return true if path has drive letter
+ */
+static bool lv_fs_path_has_drive(const char * path)
+{
+    return ('A' <= path[0] && path[0] <= 'Z') && path[1] == ':';
```

Comment:
Check the string length too
```suggestion
    if(path[0] == '\0') return false;
    return ('A' <= path[0] && path[0] <= 'Z') && path[1] == ':';
```